src/test/java/EventTest.java

```diff
@@ -0,0 +1,24 @@
+import duke.parser.Event;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EventTest {
+    Event event = new Event("Meeting", "19/9/2019");
+
+    @Test
+    public void EventGetDescriptionTest() {
+        assertEquals(event.getDescription(), "Meeting");
```

Comment:
assertEquals puts the expected output as the first argument and actual output the second :)

Reply:
Thanks for the enlightenment, actually I haven't really understood how assertEquals works